specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-04-04/kustoPool.json

```diff
@@ -0,0 +1,594 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SynapseManagementClient",
+    "version": "2021-04-04"
```

Comment:
this is a new resource type. Dont you want to release it in preview api version first?